releasetasks/templates/firefox/partner_repacks.yml.tmpl

```diff
@@ -42,6 +43,50 @@
 
         extra:
             {{ common_extras(taskname=buildername, locales=["null"], platform="null") | indent(12)}}
+
+-
+    taskId: "{{ stableSlugId(buildername_eme_free) }}"
+    reruns: 5
+    requires:
+        {% for upstream_builder in artifact_completes_builders %}
+        - {{ stableSlugId(upstream_builder) }}
+        {% endfor %}
+    task:
+        provisionerId: "buildbot-bridge"
+        workerType: "buildbot-bridge"
+        created: "{{ now }}"
+        deadline: "{{ now.replace(days=4) }}"
+        expires: "{{ never }}"
+        priority: "high"
+        retries: 5
+        scopes:
+            - project:releng:buildbot-bridge:builder-name:{{ buildername }}
```

Comment:
We are going to reuse the same buildbot buildername, just different properties. I can split variable name to something like `buildbot_buildername` and the rest.